src/components/flyout/deviceDetailFlyout.js

```diff
@@ -458,12 +464,21 @@ class DeviceDetailFlyout extends Component {
       rowData: this.state.alarmRowData,
       pagination: false
     }
+    const deviceTypeMappings = {
+       Chiller: ChillerSvg,
+       Elevator: ElevatorSvg,
+       Engine: EngineSvg,
+       Prototyping: PrototypingDeviceSvg,
+       Truck: TruckSvg
+    };
+    const deviceIconType = device.Properties.Reported.Type;
+    const svg = deviceTypeMappings[deviceIconType] || DeviceIconSvg;
     return (
       <div className="device-detail-flyout">
         <div className="device-detail-tile">
           <div className="device-detail">
             <div className="device-icon">
-              <img src={DeviceIcon} height="72" alt={`${DeviceIcon}`} />
+              <img src={svg} className="device-renderer-icon" alt={`${svg}`} />
```

Comment:
`alt` value is incorrect. This will make the alt value the path to the filename, which isn't helpful to the human reader.